scripts/install_environment.sh

```diff
@@ -130,7 +130,7 @@ function install_runtime_container() {
 
 function install_packages() {
     echo "Installing dnf packages"
-    sudo dnf install -y make python3 python3-pip git jq bash-completion xinetd
+    sudo dnf install -y make python3.9 python3-pip python39-devel git jq bash-completion xinetd
```

Comment:
bumping python is problematic on some of the versions we currently support

Reply:
It's not needed anymore. I eventually used the test infra container instead of creating virtual environment